package.json

```diff
@@ -1,25 +0,0 @@
-{
-  "name": "@ahrefs/bs-react-intl-ppx",
-  "version": "0.1.1",
```

Comment:
Where should versions be tracked?

Reply:
https://github.com/search?q=repo%3Aahrefs%2Fbs-react-intl-ppx%200.1.1&type=code

The files where it was mentioned were removed (but HISTORY.md should be update)

Reply:
In opam, versions are added when opening the PR against opam repository, e.g. https://github.com/ocaml/opam-repository/pull/23850/files#diff-6bf6c93a7f51115b310f349270eb5f98a80e15624051ac01ba3ba495083f0807R1 (see the opam file name is `melange.1.0.0/opam`).

> but HISTORY.md should be update

👍 yeah I'd add a note there under "Unreleased" section to indicate when the migration to opam & melange happened.

Reply:
Ah, TIL! Thanks.

Updated HISTORY.md:
https://github.com/ahrefs/bs-react-intl-ppx/blob/9e7418251f817c0a44d47820f923285702de54dd/HISTORY.md?plain=1#L3-L5